.build/parent-pom-template.xml

```diff
@@ -1049,6 +1049,11 @@
         <artifactId>big-math</artifactId>
         <version>2.3.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.openjdk.nashorn</groupId>
```

Comment:
@jacek-lewandowski  can we really just add it like that?

Reply:
this is only the dependency management - we declare dependency versions here. The dependency in fact is defined in [build-resolver.xml](https://github.com/apache/cassandra/pull/2434/files#diff-b7d8cc9a23d28a1713d5242f83da30838851d0685528abdb937afe7d9b899de6R137-R146)